### PR TITLE
Saw and Insolator Fixes

### PR DESCRIPTION
**Pull request by MuteTiefling**
Cleaning up Sawing recipes to get rid of log spam errors.

Palo Verde is meant to cut to Birch. Exception created in Constants to handle that

Moved thermal saw recipes over to new recipe handler

Moved IE sawing to new recipe handler

Moved Mek sawing to new recipe handler

Unified mek sawdust to EE sawdust

moved all insolators into a single file. redid entire recipe structure to work with new recipe handler

Note: I cannot get any of the quark trees to work other than the yellow one. I'm probably just too tired right now to figure it out. Maybe have a look. Everything else appears to be working.

### CODE REVIEW

File: kubejs/server_scripts/enigmatica/kubejs/base/recipes/remove.js [hunk 1]
```diff
@@ -1,100 +1,106 @@
...
+        mod: 'buildersaddition',
+        type: 'minecraft:crafting_shaped'
+    });
+
+    event.remove({
+        output: 'mekanism:sawdust',
+        mod: 'mekanism',
+        type: 'mekanism:sawing'
+    });
+
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipes/remove.js:92]: Was this needed? I already replaced mekanism:sawdust with the EE kind, I thought.

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipes/remove.js:92]: the base mekanism sawing was still outputting mekanism dust, yeah.

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipes/remove.js:92]: Damn :(

**Review by NillerMedDild — APPROVED** [review]

**Comment by NillerMedDild** [comment]
Looks good, great work :) 
I don't see any issues with the Quark sapling part of the scripts, I wonder what's going on there.

**Comment by MuteTiefling** [comment]
*sigh* maybe they'll just magically work for you.

**Comment by NillerMedDild** [comment]
They did not :/

**Comment by MuteTiefling** [comment]
Alright, I'll prep that in a little bit. I have this sneaking suspicion, however, that they're being ignored due to duplicate inputs. The one that works (yellow) is the first in the list.

**Comment by NillerMedDild** [comment]
I tried changing chances and nothing changed sadly.

**Comment by NillerMedDild** [comment]
Could be duplicate recipe name causing the issue.

**Comment by MuteTiefling** [comment]
Yeah, looks like you're right. the dynamic name is being generated only based on the primary output.... It's doing the same for the Simple Farming fruit trees as well. 

Data packs it is!

**Pull request merged**